src/poparser.js

```diff
@@ -4,18 +4,31 @@ import { Transform } from 'readable-stream';
 import util from 'util';
 
 /**
- * Po parser options
+ * @typedef {import('stream').Stream.Writable} WritableState
```

Comment:
Why not to create a base Option and then extend that type to `PoParserOptions` and `MoParserOptions`?

Reply:
I'll look into that, I hadn't taken the time to compare them.

Reply:
I looked into it, but the `Parser` in `moparser.js` doesn't take an options object. Can you clarify what you mean?

Reply:
OK I thought moParser had an options object like poParser has, and I suggested to standardise (but actually as you say MoParser does not have it and only has one string so the issue does not apply).

the one that is (maybe) slightly different is 
`{TransformOptions & {initialTreshold?: number;}} transformOptions`

that is the options set for the stream ([here](https://github.com/smhg/gettext-parser/blob/6b3b940741e46564e199078876ed60fc4ea9747b/src/poparser.js#L571)) but adding with `&` solves the issue for the moment